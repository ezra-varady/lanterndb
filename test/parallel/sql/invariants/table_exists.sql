SELECT EXISTS (
    SELECT FROM 
        pg_tables
    WHERE  
        tablename  = 'small_world'
    );
