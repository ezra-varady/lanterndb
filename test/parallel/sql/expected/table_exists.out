SELECT EXISTS (
    SELECT FROM 
        pg_tables
    WHERE  
        tablename  = 'small_world'
    );
 exists 
--------
 t
(1 row)

